src/gsy_e/gsy_e_core/rq_job_handler.py

```diff
@@ -17,12 +19,16 @@
 log = logging.getLogger()
 
 
-def decompress_and_decode_queued_strings(queued_string):
+def decompress_and_decode_queued_strings(queued_string: bytes) -> Dict:
+    """Decompress and decode data sent via redis queue."""
     return pickle.loads(decompress(queued_string))
 
 
-def launch_simulation_from_rq_job(scenario, settings, events, aggregator_device_mapping,
-                                  saved_state, job_id):
+# pylint: disable=too-many-arguments, too-many-locals
```

Comment:
Note that this will disable these warnings on the entire module, from this point onwards.